codecov.yml

```diff
@@ -63,6 +63,7 @@ ignore:
   - "service/history/workflow/errors.go"
   - "service/history/service.go"
   - "service/matching/service.go"
+  - "service/matching/tasklist/testing.go"
```

Comment:
FYI I'm adding this to internal sonarqube exclusion file

Reply:
@shaddoll what was the rationale for pulling this into a non `_test.go` file? Nit comment, but this means it's compiled into prod binaries

Reply:
Synced offline, we need to export the type, because it's used by other packages.